app/models/comment.rb

```diff
@@ -0,0 +1,13 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
```

Comment:
- [ ] [OPTIONAL] I strongly recommend that you remove this particular association as the comment has already belonged to the `:user` which is perfect.